sql/create-phpgiftregdb.sql

```diff
@@ -1,3 +1,9 @@
+CREATE DATABASE giftreg;
+
+USE giftreg;
+
+GRANT ALL ON giftreg.* TO giftreg;
```

Comment:
I would like to narrow scope so the `giftreg` user can only do DML not DDL. If the app becomes compromised, we wouldn't want this user taking over the server, even if it's in a container.